src/web/jdv/server/jaialog_store.py

```diff
@@ -107,6 +88,37 @@ def get_title_from_path(path):
 UTIME_PATH = 'goby::health::report/goby.middleware.protobuf.VehicleHealth/_utime_'
 
 
+@dataclass
+@dataclass_json
+class LogDescription:
+    '''Metadata pertaining to a log'''
+
+    bot: str
+    fleet: str
+    
+    filename: str
+    '''File stem for this log (without path, .goby or .h5 extension)'''
```

Comment:
I prefer docstring, because the IDE recognizes and shows them when you hover over the property of any object of that class.  Will do whatever we decide, though!

Reply:
Oh cool! I think it makes a lot of sense to use the docstrings for properties then